app/views/steps/completion/shared/_separator.html.erb

```diff
@@ -1,4 +1,4 @@
-<br />
+<span class="line-break-above"></span>
```

Comment:
I would try to add a class from the `utilities.scss` file here onto the element below rather than adding in a `<span>`